examples/with-react-ga/components/Layout.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
+import Router from 'next/router'
 import { initGA, logPageView } from '../utils/analytics'
 
+Router.onRouteChangeComplete = (url) => {
+  logPageView()
+}
+
```

Comment:
You're correct @vinaypuppal. It might be cleaner to use `onRouteChangeComplete` though 🤔